content/events/2021-04-29-developer-call-real-time-communications-on-urbit.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Developer Call: Real-Time Communication on Urbit"
+date = 2021-04-29
+[extra]
+pinned = false
+host = "Josh Lehman (~wolref-podlex)"
+time = "09:00 PST"
+location = "Online Event"
+image = ""
+registration_url = "https://www.meetup.com/urbit-sf/events/277674273/"
+guests = "Edward Amsden (~ritpub-sipsyl)"
+description = "Video chat on Urbit? It's nearly here with WebRTC."
+youtube = "XFct6JZHPC0"
++++
+
+Video chat on Urbit? It's nearly here using the WebRTC protocol. Edward Amsden (~ritpub-sipsyl) will be giving us a look at the WebRTC Airlock library that he has developed that handles all the hard parts of WebRTC. This will allow developers to simply call or answer calls by ship or app name. Edward will also be giving us a live text-based demo of WebRTC on urbit. Please join us for an exciting presentation.
```

Comment:
```suggestion
Video chat on Urbit? It's nearly here using the WebRTC protocol. Edward Amsden (`~ritpub-sipsyl`) will be giving us a look at the WebRTC Airlock library that he has developed that handles all the hard parts of WebRTC. This will allow developers to simply call or answer calls by ship or app name. Edward will also be giving us a live text-based demo of WebRTC on urbit. Please join us for an exciting presentation.
```